Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM circleci/openjdk:8-jdk
+
+
+USER root
+
+RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.zip && unzip google-cloud-sdk.zip && rm google-cloud-sdk.zip
```

Comment:
The `sbt-appengine` plugin only needs the App Engine SDK, not the Google Cloud SDK, for building. However, we may want to keep the Google Cloud SDK in this image if we start using it from `deploy.sh`.

Reply:
Greg Polumbo and I chatted about docker images for independent services. It may be that we want to have 2: one for building (used by CircleCI) and one for running (used by FiaB). We can probably prune this image a little but I say we wait until we make the image for FiaB which may want some of the pieces in here.

Also, https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu has some suggestions for installing the SDK into a docker image.